config/targets.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-env node */
+module.exports = {
+  browsers: [
+    'ie 9',
+    'last 1 Chrome versions',
```

Comment:
Can I ask what it is doing? Are we actually testing against some browsers other than Phantom?